docs/run.js

```diff
@@ -31,30 +28,37 @@ const scalingFactor = 100;
 let resolvedScoreAsset;
 
 /**
- * Sets {@link scorePromise} to the {@link ee.FeatureCollection} with
- * id {@link getScoreAssetPath}, or, if that does not exist, with id
- * {@link getBackupScoreAssetPath}.  Sets {@link resolvedScoreAsset} to the id
- * of the found collection.
+ * Seeds {@link Promise} cache with `eeAsset`, and returns {@link Promise} whose
```

Comment:
super nit: could do {@code eeAsset} to make more clear it's not just the string 'eeAsset' (already pretty clear just took a second read through)